frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -297,16 +298,19 @@ export const transformTopologyData = (
    * form data model specific to knative resources
    */
   const getKnativeTopologyData = (knativeResources: K8sResourceKind[], type: string) => {
-    if (knativeResources && knativeResources.length) {
+    const activeAppKnResource =
+      type !== nodeType.Revision
```

Comment:
```suggestion
      type !== NodeType.Revision
```
Enum should be in PascalCase